.gitignore

```diff
@@ -5,7 +5,8 @@
 db.json
 dist/
 node_modules/
-src/hexo/source/about/
-src/hexo/source/docs/contributor-guide
-src/hexo/source/docs/user-guide
+src/content/about/
```

Comment:
A lot of `src/content/about/*` are committed.

Reply:
This should be fixed